tests/test_dataframe_reader_extensions.py

```diff
@@ -1,10 +1,34 @@
 import pytest
-from snowflake.snowpark import Session, Row
+from snowflake.snowpark import Session, Row, DataFrameReader
 from snowflake.snowpark.types import *
+from snowflake.snowpark.dataframe import _generate_prefix
 import snowpark_extensions
 
 def test_load():    
     session = Session.builder.from_snowsql().getOrCreate()
+    cases = session.read.load(["./tests/data/test1_0.csv","./tests/data/test1_1.csv"],
+                        schema=get_schema(),
+                        format="csv", 
+                        sep=",",
+                        header="true")
+    assert 10 == len(cases.collect())
+
+def test_csv():
```

Comment:
I think we should covert more scenarios for testing. a few examples:
We have loading with schema, what about loading with schema. 
With and without header.
With different settings in the params

Reply:
please have a look to this parametrize testing mode.
https://docs.pytest.org/en/latest/example/parametrize.html#paramexamples